axelor-account/src/main/resources/domains/MoveTemplate.xml

```diff
@@ -8,8 +8,8 @@
   <entity name="MoveTemplate" lang="java" cacheable="true">
     <string name="name" title="Name" required="true"/>
     <string name="code" title="Code" required="true"/>
-    <many-to-one name="moveTemplateType" ref="MoveTemplateType" title="Type"
-      required="1"/>
+    <integer name="moveTemplateType" title="Type"
+      selection="account.move.template.type.select" required="1"/>
```

Comment:
please rename moveTemplateType to moveTemplateTypeSelect to follow our convention